server/server/models/events.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import Column, DateTime, String, Integer, func, ForeignKey
+from sqlalchemy.orm import relationship
+from . import Base
+
+
+class Event(Base):
+    """SQLAlchemy model for table events"""
+    __tablename__ = 'events'
+    
+    id = Column(Integer, primary_key=True)
+    name = Column(String, nullable=False, unique=True)
+    lng = Column(String, nullable=False)
+    lnt = Column(String, nullable=False)
```

Comment:
I assume it's latitude/longitude coordinates, thus I'd use more common abbreviations as `lat`/`long`. Also coordinates need to be stored in some numeric type like `Float`.